ConorMFinn/CO600-SeriousCyberSec
# Created room for prototype

**Description of work.**

Created a room (the hall) for the prototype room

**To test:**

Check that the room is open and uses the correct wall and floor tiles

Fixes #34
